### DISCUSSION TITLE
Mac Pro 5.1 suitable

### DISCUSSION THREAD

**Discussion by serche1512** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#198] [Q&A]
Is it possible to install the drivers into a MacPro 5.1 that has been updated to Monterrey trough Open Core?Problems?
Thx

**Comment by chris1111** [comment]
to Monterrey trough Open Core?Problems? what does it mean?

**Reply by serche1512** [reply]
Hi chris,
I read somewhere that the drivers are only for hackintosh. I have a Mac Pro that thx to Open Core, is updated up to Mac Os Monterrey. I mean if this driver can be installed also into a real Mac with no problems.
Thx!

**Comment by chris1111** [comment]
Of course it will work on real mac if the adapter is support

**Reply by serche1512** [reply]
Great, thx for your answer!
